department-of-veterans-affairs/abd-vro
# Add .bak suffix to dependency files for domain-rrd

## What was the problem?

Snyk was continuing to scan dependencies in files under `domain-rrd` and raising SecRel issues 

Associated tickets or Slack threads:


## How does this fix it?[^1]
Adds a `.bak` suffix to all `build.gradle` and `requirements.txt` files under `domain-rrd` to exclude them from Snyk scanning.

## How to test this PR
- SecRel not throwing errors related to dependencies given in files under `domain-rrd`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
